packages/design-system/src/components/Tooltip/index.ts

```diff
@@ -1,2 +1,4 @@
+export * from './Tooltip';
+export * from './TooltipIcon';
```

Comment:
We export everything mostly to get access to the type definitions. I can tighten this export statement to just reveal the types if folks think that is a better idea?